packages/client/src/components/TransactionList.js

```diff
@@ -134,50 +28,78 @@ function TransactionDetails({ safeOwners, transaction, onClose }) {
   );
 }
 
+function getTransferType(intent) {
+  if (intent.includes("from the treasury")) {
+    return "Sent";
+  }
```

Comment:
I don't love this but I'm not sure a better way to differentiate between in/out. It's not as easy as checking if `recipient === treasuryAddr` since the recipient could be the wallet address of the user that owns the treasury.